# ====== DO NOT MODIFY! AUTOGENERATED FROM AN META MESSAGE DEFINITION ======

uint8 type
uint32 id
mrta_vc/Location location
string name
uint8 hierarchy_level


string login_name			# Only users use this parameter.
mrta_vc/Computer computer

